internal/git/git.go

```diff
@@ -61,79 +58,11 @@ func Sign(ctx context.Context, rekor rekor.Writer, ident *fulcio.Identity, data
 	return sig, cert, nil
 }
 
-type VerificationSummary struct {
-	// Certificate used to sign the commit.
-	Cert *x509.Certificate
-	// Rekor log entry of the commit.
-	LogEntry *models.LogEntryAnon
-	// List of claims about what succeeded / failed during validation.
-	// This can be used to get details on what succeeded / failed during
-	// validation. This is not an exhaustive list - claims may be missing
-	// if validation ended early.
-	Claims []Claim
-}
-
-// Claim is a k/v pair representing the status of a given ClaimCondition.
-type Claim struct {
-	Key   ClaimCondition
-	Value bool
-}
-
-type ClaimCondition string
-
-const (
-	ClaimValidatedSignature  ClaimCondition = "Validated Git signature"
-	ClaimValidatedRekorEntry ClaimCondition = "Validated Rekor entry"
-)
-
-func NewClaim(c ClaimCondition, ok bool) Claim {
-	return Claim{
-		Key:   c,
-		Value: ok,
-	}
-}
-
-func Verify(ctx context.Context, rekor rekor.Verifier, data, sig []byte, detached bool) (*VerificationSummary, error) {
-	claims := []Claim{}
-
-	root, err := fulcioroots.Get()
-	if err != nil {
-		return nil, fmt.Errorf("getting fulcio root certificate: %w", err)
-	}
-	intermediates, err := fulcioroots.GetIntermediates()
-	if err != nil {
-		return nil, fmt.Errorf("getting fulcio intermediate certificates: %w", err)
-	}
-
-	cert, err := git.VerifySignature(data, sig, detached, root, intermediates)
-	if err != nil {
-		return nil, err
-	}
-	claims = append(claims, NewClaim(ClaimValidatedSignature, true))
-
-	commit, err := objectHash(data, sig)
-	if err != nil {
-		return nil, err
-	}
-
-	tlog, err := rekor.Verify(ctx, commit, cert)
-	if err != nil {
-		return nil, fmt.Errorf("failed to validate rekor entry: %w", err)
-	}
-	claims = append(claims, NewClaim(ClaimValidatedRekorEntry, true))
-
-	return &VerificationSummary{
-		Cert:     cert,
-		LogEntry: tlog,
-		Claims:   claims,
-	}, nil
-}
-
 type encoder interface {
 	Encode(o plumbing.EncodedObject) error
 }
 
-func objectHash(data, sig []byte) (string, error) {
+func ObjectHash(data, sig []byte) (string, error) {
```

Comment:
Should we just move `ObjectHash` etc as unexported methods into `pkg/git` too? Is there a benefit to keeping them exported+internal instead?

Reply:
no benefit, i was trying to keep changes as minimal as possible is all : ) I'll move it

Reply:
moved it